### ISSUE TITLE
Full-width Digits

### ISSUE THREAD

**Issue by lostguru** [DakuTree/manga-tracker#346]
I guess this is technically a feature request rather than a bug?

Some dude uploaded a manga using full-width numbering: https://mangadex.org/manga/25714
![image](https://user-images.githubusercontent.com/3195451/39086511-7b02a9ee-4560-11e8-9bd4-7d064333cf53.png)

This spits back an "ERROR: Unable to update?" when trying to follow the series.

**Comment by Vesnyx** [comment]
The actual chapter seems to have been fixed as that numbering scheme doesn't follow MangaDex's naming scheme. I don't think we're gonna parse chapters like that at any point and now chapters should be released from MangaDex like that without being corrected, so I think it'll be fine.

**Comment by DakuTree** [comment]
I agree that we shouldn't parse chapters like this, but we should probably also not throw errors for it either.  

Going to look into allowing it, but just converting it on the backend to half-width characters.
